### ISSUE TITLE
Installing bigchainDB on LXD: fast and quick

### ISSUE THREAD

**Issue by Kentoseth** [bigchaindb/bigchaindb#525]
Install LXD (instructions here): [LXD Install](https://linuxcontainers.org/lxd/getting-started-cli/)

(assumption that you are using 14.04 like bigchainDB docs specify)

`lxc launch ubuntu:14.04 my-ubuntu-cont`

(ubuntu:14.04 - this is the remote server it fetches the image from)
(my-ubuntu-cont - is the name of the container)

Here is my `install.sh`:

```
#!/bin/bash
set -ex
export DEBIAN_FRONTEND=noninteractive
apt-get install -y wget
source /etc/lsb-release && echo "deb http://download.rethinkdb.com/apt $DISTRIB_CODENAME main" | sudo tee /etc/apt/sources.list.d/rethinkdb.list
wget -qO- https://download.rethinkdb.com/apt/pubkey.gpg | sudo apt-key add -
apt-get update
apt-get install -y rethinkdb python3-pip
pip3 install --upgrade pip wheel setuptools
pip install ptpython bigchaindb
```

Copy/Paste the above `install.sh` into the directory/path you are going to execute your LXD commands from (ie. the host).

Now, from the host (and the correct directory), run this:

`cat bigchain.sh | lxc exec my-ubuntu-cont /bin/bash`

If you followed the commands correctly, you will have successfully created an LXC container (using LXD) that can get you up and running in bigchainDB in <5 minutes (depending on your internet speed).

**Comment by ttmc** [comment]
Thanks @Kentoseth 

If someone does a web search for 'BigchainDB LXC' or 'BigchainDB LXD' this issue will probably come up and be a great help.

**Comment by Kentoseth** [comment]
@ttmc 

Which part of the docs can I add it to?

It would be much easier to find within the documentation as compared to a github issue.

If you can point me in the right direction, I can write some documentation for it.

**Comment by ttmc** [comment]
Hi @Kentoseth 

You could add it as a new page in the Appendices. I just moved the page about running BigchainDB in Docker to the Appendices (in PR #532). Here's how:

1. Write a Markdown file with the instructions named `run-with-lxd.md` and put it in `bigchaindb/docs/source/appendices/`
2. Add a line to the file `bigchaindb/docs/source/appendices/index.rst` that says `   run-with-lxd` (with no .md on the end)

There are further details about how to contribute to BigchainDB in [our CONTRIBUTING.md file](https://github.com/bigchaindb/bigchaindb/blob/master/CONTRIBUTING.md).

**Comment by Kentoseth** [comment]
PR opened: #537

**Comment by ttmc** [comment]
Resolved by #537
